pkg/controllers/scheduler/framework/runtime/framework.go

```diff
@@ -246,9 +247,13 @@ func (f *frameworkImpl) RunSelectClustersPlugin(
 				result.Message(),
 			)
 			klog.Error(msg)
-			return clusters, framework.NewResult(framework.Error, msg)
+			return nil, framework.NewResult(framework.Error, msg)
 		}
-		return clusters, result
+		clusterScores = newClusterScores
```

Comment:
If `MaxClusters` doesn't sort internally, the framework needs to do the sorting. Otherwise in the absence of `PrioriotySort`, `MaxClusters` will stop working correctly.

Reply:
A plugin shouldn't depend on another plugin to function.

Reply:
I got your points. There will still be some depency if we want multiple select plugins. For example, priority must be before maxClusters. Let's think about refacter the framework later.